### PR TITLE
First pass at a refresh

### PR DESCRIPTION
**Pull request by bnb**
First pass at a refresh for NodeSchool NYC! 🎉

This includes a few things:

- `.gitignore`: so `node_modules` doesn't get checked in
- `LICENSE`: Just puts the license in `package.json` as a file (lowers the barrier to contribution for some contributors)
- `README.md`: Refreshes the README.md with the current state of things as I know it
- Website: Adds an updated website, tweaked from the NodeSchool San Francisco site

There are a few concepts that I assumed we'd adopt from NodeSchool San Francisco or my best guess that are included but we aren't _required_ to adopt:

- GitHub as a central means of organization
- Focused Session(s) as an additional part of the event
- Meeting time: Listed as 4:00pm, with the mentor meeting starting at 3:30 – going until 7pm
- Food: Pizza as the default. At least at 11 Times Square, Subway is also an option!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,42 @@
...
+- **Franziska Hinkelmann:**: [GitHub](https://github.com/fhinkel), [Twitter](https://twitter.com/fhinkel)
+- **Joe Sepi**: [GitHub](https://github.com/joesepi), [Twitter](https://twitter.com/joe_sepi/)
+- **Tierney Cyren**: [GitHub](https://github.com/bnb), [Twitter](https://twitter.com/bitandbang/)
+- **John Elliott**: [GitHub](https://github.com/johnelliott)
 
-[See Upcoming Events](https://github.com/nodeschool/nyc/blob/master/upcoming-events.md)
+## Contributing to the website
 
----
+The website is generated using Mustache templates, Stylus, and JavaScript. It runs on [GitHub Pages](https://pages.github.com/).
```
Comment (fhinkel) [inline README.md]: Should this include a sentence that we accept PRs?

Comment (bnb) [reply README.md]: Done!

**Comment by bnb** [comment]
Going to merge this at/after ~4pm ET so we can get the ball rolling!

**Comment by johnelliott** [comment]
I updated my info and the markdown file with upcoming events.

**Comment by bnb** [comment]
Going to merge 👍

**Pull request merged**